src/bytes.rs

```diff
@@ -1463,6 +1475,21 @@ impl Inner {
         }
     }
 
+    /// Insert a byte into the next slot and advance the len by 1.
+    #[inline]
+    unsafe fn put_u8(&mut self, n: u8) {
```

Comment:
Why is this function unsafe? While the implementation requires `unsafe`, I don't think that unsafety leaks out?